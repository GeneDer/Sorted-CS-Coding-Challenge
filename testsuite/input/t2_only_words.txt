,j:u]ly ap_ple*^ cat gen!@#e bi?rd fly i@c%e elec!t~r+ic do@g ha)nds
