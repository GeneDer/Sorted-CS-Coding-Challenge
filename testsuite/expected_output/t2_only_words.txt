apple bird cat dog electric fly gene hands ice july